ros/angel_system_nodes/angel_system_nodes/intent_to_command.py

```diff
@@ -0,0 +1,134 @@
+import rclpy
+from rclpy.node import Node
+import yaml
+
+from angel_msgs.msg import (
+    InterpretedAudioUserIntent,
+    SystemCommands,
+)
+
+
+class IntentToCommand(Node):
+    """
+    ROS node that converts user intent messages to system commands.
+    """
+
+    def __init__(self):
+        super().__init__(self.__class__.__name__)
+
+        self._explicit_intent_topic = (
+            self.declare_parameter("explicit_intent_topic", "")
+            .get_parameter_value()
+            .string_value
+        )
+        self._confirmed_intent_topic = (
+            self.declare_parameter("confirmed_intent_topic", "")
+            .get_parameter_value()
+            .string_value
+        )
+        self._sys_cmd_topic = (
+            self.declare_parameter("sys_cmd_topic", "")
+            .get_parameter_value()
+            .string_value
+        )
+        self._intent_to_cmd_map_path = (
+            self.declare_parameter("intent_to_cmd_map", "")
+            .get_parameter_value()
+            .string_value
+        )
+
+        if self._explicit_intent_topic == "":
+            raise ValueError(
+                "Please provide the explicit intent topic with the `explicit_intent_topic` parameter"
+            )
+        if self._confirmed_intent_topic == "":
+            raise ValueError(
+                "Please provide the confirmed intent topic with the `confirmed_intent_topic` parameter"
+            )
+        if self._intent_to_cmd_map_path == "":
+            raise ValueError(
+                "Please provide the intent command map path with the `intent_to_cmd_map` parameter"
+            )
+        if self._sys_cmd_topic == "":
+            raise ValueError(
+                "Please provide the system command topic with the `sys_cmd_topic` parameter"
+            )
```

Comment:
I attempted to be more concise about this in my other branch. Would it make sense to make use of that here, or do you think that is extraneous?

Reply:
Did that alternative approach seem any easier/better to you? I want to get more than my thoughts into the mix.